src/app/components/Delegations/index.tsx

```diff
@@ -1,45 +1,63 @@
 import { FC } from 'react'
 import { useTranslation } from 'react-i18next'
-import { Table, TableCellAlign, TableColProps } from '../../components/Table'
-import { Delegation } from '../../../oasis-nexus/api'
+import { DebondingDelegation, Delegation } from '../../../oasis-nexus/api'
+import { Table, TableCellAlign, TableColProps } from '../Table'
 import { TablePaginationProps } from '../Table/TablePagination'
 import { RoundedBalance } from '../RoundedBalance'
-import { ValidatorLink } from '../Validators/ValidatorLink'
+import { AccountLink } from '../Account/AccountLink'
 
 type DelegationsProps = {
-  delegations?: Delegation[]
+  debonding?: boolean
+  delegations?: Delegation[] | DebondingDelegation[]
   isLoading: boolean
   limit: number
   pagination: false | TablePaginationProps
 }
 
-export const Delegations: FC<DelegationsProps> = ({ delegations, isLoading, limit, pagination }) => {
+export const Delegations: FC<DelegationsProps> = ({
+  debonding,
+  delegations,
+  isLoading,
+  limit,
+  pagination,
+}) => {
   const { t } = useTranslation()
 
   const tableColumns: TableColProps[] = [
-    { key: 'name', content: t('validator.title') },
-    { align: TableCellAlign.Right, key: 'shares', content: t('validator.shares') },
-    { align: TableCellAlign.Right, key: 'staked', content: t('common.staked') },
+    { key: 'delegator', content: t('common.address') },
+    { key: 'amount', content: t('validator.amount'), align: TableCellAlign.Right },
+    { key: 'shares', content: t('validator.shares'), align: TableCellAlign.Right },
+    ...(debonding
+      ? [{ key: 'debondingEnd', content: t('validator.debondingEnd'), align: TableCellAlign.Right }]
+      : []),
   ]
   const tableRows = delegations?.map(delegation => ({
-    key: delegation.validator,
+    key: delegation.delegator,
     data: [
       {
-        // TODO: Use trimmed name when API is updated
-        content: <ValidatorLink address={delegation.validator} alwaysTrim network={delegation.network} />,
-        key: 'name',
+        content: <AccountLink scope={delegation} address={delegation.delegator} />,
```

Comment:
In Account details view we need to show ValidatorLink with delegation.validator instead of AccountLink